proposals/1.2-proposal.md

```diff
@@ -0,0 +1,113 @@
+STRT.M           635.0000:
+STOP.M           400.0000:
+STEP.M            -0.1250:
+NULL.             -999.25:
+COMP.             COMPANY:      ANY OIL COMPANY INC.
+WELL.                WELL:      ANY ET AL A9-16-49-20
+FLD .               FIELD:      EDAM
+LOC .            LOCATION:      A9-16-49-20W3M
+PROV.            PROVINCE:      SASKATCHEWAN
+SRVC.     SERVICE COMPANY:      ANY LOGGING COMPANY INC.
+DATE.            LOG DATE:      13-DEC-86
+UWI .      UNIQUE WELL ID:      100091604920W300
+-----------------------------------------------------------------------------
+```
+**Proposed Data Struct- Table with MNEM, unit data type and Info as columns**
+
+~C (Curve Information)
+- This basically contains all type of measurements of the particular well that were taken, ie, contains the list of all the possible values that are there in the data table.
+- By possible values, it is meant that type of measurements like - Depth, Bulk Density, etc.
+- The curves described in this section must be present in the data set.
+```
+-----------------------------------------------------------------------------
+~Curve Information Section
+#MNEM.UNIT	API CODE	Curve Description	
+#---------    -------------   -------------------------------
+DEPTH.M                      :    1       DEPTH
+RHOB .K/M3        7 350 02 00:    2       BULK DENSITY
+NPHI .VOL/VOL     7 890 00 00:    3       NEUTRON POROSITY - SANDSTONE
+MSFL .OHMM        7 220 01 00:    4       Rxo RESISTIVITY
+SFLA .OHMM        7 222 01 00:    5       SHALLOW RESISTIVITY
+ILM  .OHMM        7 120 44 00:    6       MEDIUM RESISTIVITY
+ILD  .OHMM        7 120 46 00:    7       DEEP RESISTIVITY
+SP   .MV          7 010 01 00:    8       SPONTANEOUS POTENTIAL
+GR   .GAPI        7 310 01 00:    9       GAMMA RAY
+CALI .MM          7 280 01 00:    10      CALIPER
+DRHO .K/M3        7 356 01 00:    11      DENSITY CORRECTION
+-----------------------------------------------------------------------------
+```
+**Proposed Data Struct - Table with MNEM, unit, API and Curve description as columns**
+
+~P (Parameter Information)
+
+-This section is **optional**. It defines the values of various parameters relating to this well.
+```
+-----------------------------------------------------------------------------
+~Parameter Information Section
+#MNEM.UNIT      Value        Description
+#---------   -------------   ------------------------------
+BHT. DEGC         24.0000:  Bottom Hole Temperature
+BS  .MM          222.0000:  Bit Size
+FD  .K/M3        999.9999:  Fluid Density
+MDEN.K/M3       2650.0000:  Logging Matrix Density
+MATR.              1.0000:  Neutron Matrix (1=Sand)
+FNUM.              1.0000:  Tortuosity  Constant Archie's (a)
+FEXP.              2.0000:  Cementation Exponent Archie's (m)
+-----------------------------------------------------------------------------
+```
+
+~O (Other Information)
+
+-This section is  optional. Its intended use is as a remarks or comments section.
+
```

Comment:
Depending on what is written in this section, we may simply use a string to save the entire comment.